system/services/rendering/ContentRendererService.cfc

```diff
@@ -75,6 +79,7 @@ component {
 			rendered = this.render(
 				  renderer = renderer
 				, data     = arguments.data
+				, record   = arguments.record
```

Comment:
I think here, we should not add a new `record` arg to the `render()` function (but ok here in the `renderField` function). Instead, make use of the `args` struct:

```
rendered = this.render(
      renderer = renderer
    , data     = arguments.data
    , args     = { objectName=arguments.object, recordId=arguments.recordId, record=arguments.record }
);
```

`render()` is more general purpose and already accepts arbitrary `args` and I think better to keep it that way.